src/AcceptanceTests/ConfigureEndpointAzureServiceBusTransport.cs

```diff
@@ -17,13 +17,16 @@ public class ConfigureAzureServiceBusTransport : IConfigureTestExecution
         typeof(AllNativeMultiQueueTransactionTransports)
         //typeof(AllNativeMultiQueueTransactionTransports)
```

Comment:
Suspect this should have been `AllTransportsWithoutNativeDeferralAndWithAtomicSendAndReceive`, which causes `When_timeout_dispatch_fails_on_timeout_data_removal` to fail.